Laboratory/Laboratory.csproj

```diff
@@ -61,4 +61,21 @@
     <ProjectReference Include="..\Element.NET\Element.NET.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
These are all added by a glob pattern in line 22

Reply:
Done 👍